README.md

```diff
@@ -1,31 +1,60 @@
 
-The current process for development of the OpenAPI Specification is described in [Development Guidelines](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/DEVELOPMENT.md).
+The OpenAPI Specification does not require you to rewrite your existing API.  It does not require binding any software to a service--the service being described may not even be yours.  It does, however, require the capabilities of the service be described in the structure of the OpenAPI Specification.  Not all services can be described by OpenAPI--this specification is not intended to cover every possible use-case of a REST API. The OpenAPI Specification does not define a specific development process such as design-first or code-first.  It does facilitate either technique by establishing clear interactions with a REST API.
+
+This GitHub project is the starting point for OpenAPI.
+Here you will find the information you need about the OpenAPI Specification, simple examples of what it looks like,
+and some general information regarding the project.
+
+## Current Version - 3.0
+
+The current version of the OpenAPI specification is [OpenAPI Specification 3.0](versions/3.0.md).
+
+### Previous Versions
+
+This repository also contains the [OpenAPI Specification 2.0](versions/2.0), which is identical to the Swagger 2.0 specification, 
+as well as the Swagger 1.2 and Swagger 2.0 specifications.
+
+Each folder in this repository, such as [examples](examples) and [schemas](schemas), should contain folders pertaining to the current and previous versions of the specification.
+
+## See it in Action
+
+If you just want to see it work, check out the [list of current examples](examples/v3.0).
+
+## Tools and Libraries
+
+Looking to see how you can create your own OpenAPI definition, present it or otherwise use it? Check out the growing
+[list of 3.0 Implementations](IMPLEMENTATIONS.md).
 
 ## Participation
 
-The OpenAPI Specification is a community driven, open project hosted by the Linux Foundation. The OAI encourages participation from individuals and companies alike. If you want to participate in the evolution of the OpenAPI Specification, consider taking the following actions:
+The current process for development of the OpenAPI Specification is described in 
+[Development Guidelines](DEVELOPMENT.md).
+Development of the next version of the OpenAPI Specification is guided by the [Technical Developer Community](https://www.openapis.org/participate/how-to-contribute/governance#TDC) and governed by the [TDC Contributors](CONTRIBUTORS.md).  This group of committers bring their API expertise, incorporate feedback from the community, and expand the group of committers as appropriate.  All development activity on the future specification will be performed as features and merged into this branch.  Upon release of the future specification, this branch will be merged to master.
```

Comment:
Can probably drop the process info about "this branch" and merging" as when this becomes the README, this will have been completed and will be in master. I think it is too early to say what the post 3.0.0 process and branch strategy will be. Need more TDC input on this. (I think removing this from the main README is OK - it should be in DEVELOPING.md which is mentioned immediately below)